site/en/docs/devtools/javascript/reference/index.md

```diff
@@ -21,8 +21,21 @@ See [Get Started With Debugging JavaScript In Chrome DevTools][1] to learn the b
 ## Pause code with breakpoints {: #breakpoints }
 
 Set a breakpoint so that you can pause your code in the middle of its execution.
+To learn how to set breakpoints, see [Pause Your Code With Breakpoints][2].
 
-See [Pause Your Code With Breakpoints][2] to learn how to set breakpoints.
+### Check values when paused {: #inline-eval }
+
+While the execution is paused, the debugger evaluates all variables, constants, and objects up to a breakpoint and shows their current values inline next to the corresponding declarations.
```

Comment:
"up to a breakpoint" is not accurate. It depends on where the user put a breakpoint. Let's say you put a breakpoint inside a function (line 20-40). The inline review won't show for other functions in line 10-19.